src/dev/pages/stack/stack.scss

```diff
@@ -3,12 +3,36 @@
 }
 
 .box {
-  padding: 36px;
   border: 4px dashed var(--mdc-theme-text-secondary-on-background);
   background-color: var(--mdc-theme-text-disabled-on-background);
```

Comment:
Let's replace any `--mdc-*` variable in these demos with the new `--forge-*` equivalents so we don't have to do it later when we fully remove MDC.